translations/remote-schemas.kannada.md

```diff
@@ -0,0 +1,83 @@
+# ರಿಮೋಟ್ ಸ್ಕೀಮಾಗಳು
+
+ನೀವು ಎಲ್ಲಾ ಪ್ರಕಾರದ ಗ್ರಾಫ್ಕ್ಯೂಎಲ್ ಅನ್ನು ಒಂದೇ ಸ್ಥಳದಿಂದ ಪ್ರಶ್ನಿಸಲು ಬಯಸಿದರೆ ನಂತರ ದೂರಸ್ಥ ಗ್ರಾಫ್ಕ್ಯೂಎಲ್ ಸ್ಕೀಮಾವನ್ನು ಗ್ರಾಫ್ಕ್ಯೂಎಲ್ ಎಂಜಿನ್‌ನ ಪೋಸ್ಟ್‌ಗ್ರೆಸ್ ಆಧಾರಿತ ಸ್ಕೀಮಾದೊಂದಿಗೆ ವಿಲೀನಗೊಳಿಸಿ. ರಿಮೋಟ್ ಸ್ಕೀಮಾಗಳು ಕೆಲವು ಉದಾಹರಣೆಗಳಿಗೆ ಸೂಕ್ತವಾಗಿವೆ:
+
+- ರೂಪಾಂತರಗಳನ್ನು ಉತ್ತಮಗೊಳಿಸುವುದು (ಉದಾಹರಣೆಗೆ ಒಳಪಡಿಸುವ ಮೊದಲು ಮೌಲ್ಯೀಕರಣ)
```

Comment:
ರೂಪಾಂತರಗಳನ್ನು provides different meaning.

Reply:
ರೂಪಾಂತರಗಳನ್ನು would provide the meaning for `transformation` (ರೂಪಾಂತರ -- to transform)
We could replace it by ಬದಲಾವಣೆ, ಪರಿವರ್ತನೆ, ಮಾರ್ಪಾಟು.
Let me know which is suitable.

Reply:
Mutations is the name of a GrapghQL operation, so it makes no sense to mention the translated word.

ಮ್ಯುಟೇಷನ್ಸ್ಗಳನ್ನು  ಕಸ್ಟಮೈಸ್ ಮಾಡುವುದು would provide exact meaning